themis/_index.md

```diff
@@ -88,7 +89,11 @@ We created Themis to build other products on top of it - i.e. [Acra](https://www
 
 1. Themis doesn't provide a low-level/raw cryptographic API for encryption and hashing.
 
-   For example, Themis doesn't provide a way to select a certain cryptographic cipher, or key length, or exact elliptic curve, or hash function (there is no API like `encrypt(cipher: AES, key-length:256, mode:GCM)`). 
+   For example, Themis doesn't provide a way to select a certain cryptographic cipher, or key length, or exact elliptic curve, or hash function.
+   There is no API like
```

Comment:
Have you seen this change in built docs?

Single `\n` doesn't work as new line in markdown, so this piece of doc is still one single line. Are you sure it looks ok?

Reply:
Yeah, I checked how Hugo renders it. The code fence (<code>```</code>) is correctly interpreted as a block that breaks the paragraph. But I'll add an empty line to avoid triggering OCD.

Reply:
Thank you!
My considerations were based on some previous formatting pieces I've seen in this doc, that _looked like_ there meant to be a line break, but Hugo didn't render it.